tests/runtime/test_runtime_metrics_api.py

```diff
@@ -43,3 +43,28 @@ def test_manually_start_runtime_metrics(run_python_code_in_subprocess):
     )
 
     assert status == 0
+
+
+def test_start_runtime_metrics_via_env_var(monkeypatch, ddtrace_run_python_code_in_subprocess):
+    """
+    When running with ddtrace-run and DD_RUNTIME_METRICS_ENABLED is set
+        Runtime metrics worker starts and there are no errors
+    """
+
+    _, _, status, _ = ddtrace_run_python_code_in_subprocess(
+        """
+from ddtrace.internal.runtime.runtime_metrics import RuntimeWorker
+assert RuntimeWorker._instance is None
+"""
+    )
+    assert status == 0
+
+    monkeypatch.setenv("DD_RUNTIME_METRICS_ENABLED", "true")
+
+    _, _, status, _ = ddtrace_run_python_code_in_subprocess(
+        """
+from ddtrace.internal.runtime.runtime_metrics import RuntimeWorker
+assert RuntimeWorker._instance is not None
+"""
+    )
```

Comment:
```suggestion
""",
    env={"DD_RUNTIME_METRICS_ENABLED": "true"},
    )
```

`ddtrace_run_python_code_in_subprocess` passed extra kwargs to `subprocess.Popen` via `call_program` https://github.com/DataDog/dd-trace-py/blob/ffa3826aa9337682dc53f75852a12034b4b7f246/tests/utils.py#L947-L950